### PR TITLE
Create jouornal_of_management_studies.csl

### PR DESCRIPTION
**Pull request by tetyanaloskutova**
Style for the Journal of Management Studies submissions

### CODE REVIEW

**Comment by rmzelle** [comment]
@tetyanaloskutova, I added most of the things @damnation333 requested. Could you tell us which style you started out from, so we can add a "template" link? (https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

**Comment by tetyanaloskutova** [comment]
Thank you, Rintze.
The template was http://www.zotero.org/styles/human-resource-management-journal.

File: journal-of-management-studies.csl [hunk 1]
```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal of Management Studies</title>
+    <title-short>JMS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-management-studies</id>
+    <link href="http://www.zotero.org/styles/journal-of-management-studies" rel="self"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1467-6486/homepage/ForAuthors.html" rel="documentation"/>
```
Comment (damnation333) [inline journal-of-management-studies.csl]: Can you add the URL after line 8 with rel="template"?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**